test/second_q/drivers/psi4d/test_driver_methods_psi4.py

```diff
@@ -0,0 +1,89 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test Driver Methods PSI4 """
+
+import unittest
+
+from test.second_q.drivers.test_driver_methods_gsc import TestDriverMethods
+from qiskit_nature.second_q.drivers import Psi4Driver
+from qiskit_nature.second_q.transformers import FreezeCoreTransformer
+import qiskit_nature.optionals as _optionals
+
+
+class TestDriverMethodsPSI4(TestDriverMethods):
+    """Driver Methods PSI4 tests"""
+
+    psi4_lih_config = """
+molecule mol {{
+   0 1
+   Li  0.0 0.0 0.0
+   H   0.0 0.0 1.6
```

Comment:
```suggestion
   H   0.0 0.0 1.6
   no_com
   no_reorient
```
Or are you handling this differently now?